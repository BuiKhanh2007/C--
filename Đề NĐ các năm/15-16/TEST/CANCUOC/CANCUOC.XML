<Exam Name="CANCUOC" TestScore="0.5" TestTimeout="1" ResultComparer="WordIgnoreCase">
  <Inputs>
    <Text>CANCUOC.INP</Text>
  </Inputs>
  <Outputs>
    <Text>CANCUOC.OUT</Text>
  </Outputs>
  <Tests>
    <Test ID="Test01" Score="0" Timeout="0" />
    <Test ID="Test02" Score="0" Timeout="0" />
    <Test ID="Test03" Score="0" Timeout="0" />
    <Test ID="Test04" Score="0" Timeout="0" />
    <Test ID="Test05" Score="0" Timeout="0" />
    <Test ID="Test06" Score="0" Timeout="0" />
    <Test ID="Test07" Score="0" Timeout="0" />
    <Test ID="Test08" Score="0" Timeout="0" />
    <Test ID="Test09" Score="0" Timeout="0" />
    <Test ID="Test10" Score="0" Timeout="0" />
  </Tests>
</Exam>