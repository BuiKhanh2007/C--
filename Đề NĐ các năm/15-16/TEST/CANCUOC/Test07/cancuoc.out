13669646246464370